l10n_it_account_stamp/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_account_config_settings">
```

Comment:
Please place `id` attribute before `model` according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#21format

Reply:
Done

Reply:
Thanks!